spec/ndx-binned-spikes.extensions.yaml

```
@@ -70,3 +70,41 @@ groups:
     neurodata_type_inc: DynamicTableRegion
     doc: A reference to the Units table region that contains the units of the data.
     quantity: '?'
+- neurodata_type_def: BinnedSpikes
+  neurodata_type_inc: NWBDataInterface
+  default_name: BinnedSpikes
+  doc: A data interface for non-aligned binned spike counts.
+  attributes:
+  - name: name
+    dtype: text
+    value: BinnedSpikes
+    doc: The name of this container
+  - name: description
+    dtype: text
+    value: Binned spike counts.
+    doc: A description of what the data represents
+  - name: bin_width_in_milliseconds
+    dtype: float64
+    doc: The length in milliseconds of the bins
```

```suggestion
  - name: bin_width_in_ms
    dtype: float64
    doc: The length in milliseconds of the bins
```

I would also make this change in `BinnedAlignedSpikes`.

---

Ok, done.